ynput/ayon-core
# Maya: add option to stripe shader assignment for setdress family

## Changelog Description
This PR is to add support to exclude shaders assignment when publishing the scene with setdress family

## Additional info
Update the maya addon to 0.2.5
You can set the setdress settings in ayon setting 
![image](https://github.com/ynput/ayon-core/assets/64118225/4923fe36-821b-4bc5-b48d-94224d6451ad)

## Testing notes:
1. Create setdress and families such as mayascene and layout
2. In setdress, you can choose whether you want to include shader 
![image](https://github.com/ynput/ayon-core/assets/64118225/4d6d8790-4674-4bd6-8f93-2fab8ec7f826)

3. Publish 
4. Should be published successfully
5. Check published setdress scene, the scene should be without shader assignment if the shader option is turned off.
6. Check the published scene from mayascene/layout family, the scene should be published with shader assignment
